src/os/windows/crashlog_win.cpp

```diff
@@ -271,6 +271,7 @@ static const uint MAX_FRAMES     = 64;
 			/* Get symbol name and line info if possible. */
 			DWORD64 offset;
 			if (proc.pSymGetSymFromAddr64(hCur, frame.AddrPC.Offset, &offset, sym_info)) {
+				sym_info->Name[MAX_SYMBOL_LEN - 1] = '\0'; // SymGetSymFromAddr64 does not null-terminate truncated symbol names, ensure that symbol name is null-terminated
```

Comment:
I wonder to what extent the mangled name might still contain non-UTF8 characters. It might be better to just pass it through `StrMakeValid` with the appropriate end of the buffer. That way broken characters for printing/visualisation in OpenTTD will be removed too.